Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -111,17 +116,40 @@ public async Task PostAsync_InvalidPost_BadRequest()
                 });
             }).CreateClient();
 
-            client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Test");
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Test");
 
-            NewPostDto newPost = new() { Content = "Valid Content" };
+            // Initialize a string, and then use a forloop to add to the string until it is one longer than the allowed post size
 
-            StringContent stringContent = new(System.Text.Json.JsonSerializer.Serialize(newPost), Encoding.UTF8, "application/json");
+            StringBuilder longPost = new StringBuilder();
+
+            for (int i = 0; i < Constants.PostMaxLength + 1; i++)
+            {
+                longPost.Append("X");
+            }
+
+            //Create New Post DTO objects that will be sent to the API.
+
+            NewPostDto invalidPostTooLong = new() { Content = longPost.ToString() };
+            NewPostDto invalidPostNoContent = new() { Content = "" };
+
+            //  Serializes request object into json format.
+
+            StringContent invalidStringTooMuchContent = new(JsonSerializer.Serialize(invalidPostTooLong), Encoding.UTF8, "application/json");
+            StringContent invalidstringNoContent = new(JsonSerializer.Serialize(invalidPostNoContent), Encoding.UTF8, "application/json");
 
             // Act
-            var response = await client.PostAsync("api/posts", stringContent);
+
+            var invalidresponseTooMuchContent = await client.PostAsync("api/posts", invalidStringTooMuchContent);
+            var invalidResponseNoContent = await client.PostAsync("api/posts", invalidstringNoContent);
 
             // Assert
-            Assert.Equal(System.Net.HttpStatusCode.BadRequest, response.StatusCode);
+            // Ensures that the valid request was created, and that posts that are too long or too short are rejected.
+
+            Assert.Equal(HttpStatusCode.BadRequest, invalidresponseTooMuchContent.StatusCode);
+            Assert.Equal(HttpStatusCode.BadRequest, invalidResponseNoContent.StatusCode);
+
+
+
```

Comment:
what's with all the blank lines

Reply:
I changed it so comments are right above what they are documenting, also removed a couple other unnecessary spaces. Fixed